server/templates.go

```diff
@@ -16,6 +17,14 @@ type page struct {
 	Config *config.Config
 }
 
+func executeTextTemplate(t *texttemplate.Template, context interface{}) ([]byte, error) {
```

Comment:
Let's reduce this duplication a little by

```
type Template {
  Execute(wr io.Writer, data interface{}) error
}
```

and making `executeTemplate` take a `Template`